docs/api/desktop-capturer.md

```diff
@@ -94,6 +102,22 @@ which can detected by [`systemPreferences.getMediaAccessStatus`][].
 [`navigator.mediaDevices.getUserMedia`]: https://developer.mozilla.org/en/docs/Web/API/MediaDevices/getUserMedia
 [`systemPreferences.getMediaAccessStatus`]: system-preferences.md#systempreferencesgetmediaaccessstatusmediatype-windows-macos
 
+### `desktopCapturer.isDisplayMediaSystemPickerAvailable()` _Experimental_ _MacOS_
+
+Returns `Boolean`, whether or not requesting desktop content via
+the system picker is supported on this platform.
+
+Currently this will only return `true` on macOS 14.4 and higher. When
+true, use the `getNativePickerSource` method to return the system picker's
+selected media stream. Not doing so may cause a warning dialog to your users
+on macOS 15 and higher.
+
+### `desktopCapturer.getNativePickerSource` _Experimental_ _Readonly_ _MacOS_
```

Comment:
```suggestion
### `desktopCapturer.isDisplayMediaSystemPickerAvailable()` _Experimental_ _macOS_

Returns `Boolean`, whether or not requesting desktop content via
the system picker is supported on this platform.

Currently this will only return `true` on macOS 14.4 and higher. When
true, use the `getNativePickerSource` method to return the system picker's
selected media stream. Not doing so may cause a warning dialog to your users
on macOS 15 and higher.

### `desktopCapturer.getNativePickerSource` _Experimental_ _Readonly_ _macOS_
```

Archaeologist failing due to the casing here.